generators/entity-server/templates/src/main/java/package/repository/EntityRepository.java.ejs

```diff
@@ -118,6 +126,19 @@ public interface <%= entityClass %>Repository extends <% if (databaseTypeSql) {
     @Query("<%- (databaseTypeMongodb) ? "{'id': ?0}" : "#{#n1ql.selectEntity} USE KEYS $1 WHERE #{#n1ql.filter}" %>")
     Optional<<%= persistClass %>> findOneWithEagerRelationships(<%= primaryKey.type %> id);
     <%_
-    }
-  } _%>
+        }
+    } _%>
+    <%_ if (databaseTypeCouchbase) { _%>
```

Comment:
I think it's better to move this to `N1qlCouchbaseRepository`, and introducing it back, this way there is no need to duplicate all those lines in every repository. This will also solve the problem with AuthorityRepository

Reply:
I've moved it into N1qlCouchbaseRepository and reintroduced it